assigner/backends/decorators.py

```diff
@@ -0,0 +1,15 @@
+from assigner.backends import GitlabBackend, MockBackend
+from assigner.config import config_context
+
+
+def require_backend(func):
```

Comment:
How about naming this `with_backend_and_config` and renaming `config_context` to `with_config`?

Reply:
I'm afraid of using 'with' in decorators, it sounds context managery. Maybe `requires_config_and_backend` and `requires_config` (I put an s in there too).

---
I spent more time looking at this than I should have. Most of the examples of decorators I look at are single words--- `patch`, `wraps`, `partial`, `classmethod`, `staticmethod`... I couldn't find more. I'm not sure if there's a wrong way to name them.

Reply:
SGTM!

Reply:
Renamed as described.